packages/webapp/src/inductions/components/pending-inductions.tsx

```diff
@@ -22,27 +26,53 @@ export const PendingInductions = ({ isActive }: Props) => {
         ? currentInductions.endorsements
         : [];
 
-    return isLoading ? (
-        <>Loading Inductions...</>
-    ) : (
-        <>
-            <Heading size={2}>Your Inductions</Heading>
+    const userEndorsements: Endorsement[] = endorsements.filter(
+        (end: Endorsement) => end.inviter !== end.endorser
+    );
+
+    if (isLoading) {
+        return (
             <div className="space-y-4">
-                {!isActive && !inductions.length && (
-                    <Text className="mb-4">
-                        Have you already reached out to your inviter? As soon as
-                        they invite you, the induction process will be displayed
-                        here.
-                    </Text>
-                )}
-                <InductionsList inductions={inductions} isInviter={isActive} />
+                <>Loading inductions...</>
+            </div>
+        );
+    }
+
+    const thereAreEndorsements = !!userEndorsements.length;
+    const thereAreInductions = !!inductions.length;
```

Comment:
nit: `x.length > 0` more readable than `!!`

Reply:
I can get on board with that! 😃